ang/civicase/activity/feed/directives/activity-month-nav.directive.js

```diff
@@ -47,6 +47,7 @@
   module.controller('civicaseActivityMonthNavController', civicaseActivityMonthNavController);
 
   function civicaseActivityMonthNavController ($rootScope, $scope, crmApi) {
+    var previousApiCalls = false;
```

Comment:
defined it as `null` instead of `false`. If you initialize it as `false` it would be implied that this variable will be a boolean value at all times.